src/main/flight/pid.c

```diff
@@ -93,6 +93,21 @@ void pidResetErrorGyro(void)
 
 const angle_index_t rcAliasToAngleIndexMap[] = { AI_ROLL, AI_PITCH };
 
+
+// Low Pass filter for PID controllers
+void pt1_element(int axis, float *delta, uint8_t f_cut, int type) {
+
+	float dT = (float)cycleTime * 0.000001f;
+
+    float    RC;
+    static float    lastDelta[][3] = {{ 0.0f, 0.0f, 0.0f }};
+
```

Comment:
Mh i tried adding the gyro to this https://github.com/henn1001/cleanflight/commit/5808926ebbfec099dceeda3111a6191d4d92f76d
Now im getting a full throttle as soon as i arm if i activate it in combination with one of the other two filters. Combining the other two filters is fine though..